srcalc/src/main/webapp/css/srcalc.css

```diff
@@ -99,6 +105,29 @@ a.btn-link:link, a.btn-link:visited {
 	text-decoration: none;
 }
 
+a.btn-em:link {
+	background: #CCCCCC;
+	font-size: 1em;
+}
+
+/* A button that is an emphasized version of the normal button.
+*/
+button.button-em {
+	color: #000;
```

Comment:
Why not just stick with the default (#222)? I don't think we need to specify a special color at all.